src/commands_discord/social/remind-command.js

```diff
@@ -105,7 +105,7 @@ class RemindCommand extends DiscordCommand {
   /**
    * Gets the default value for a given argument definition.
    * Used when unable to scan the argument from the command's text.
-   * @param  {Message}        message the command's message
+   * @param  {BaseMessage}        message the command's message
    * @param  {CommandArgDef}  arg     the argument definition
    * @return {Object}                 the default value
    */
```

Comment:
This function is broken, because you call "channel.id" on the BaseMessage instead of the originalMessage. Please check "getDefaultDiscordArgValue" function in all commands and change "message" to "message.originalMessage" where needed.